### PR TITLE
Jaclyn taroni/2023 02 19 edits

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
Stacked on #238 

Summarizing changes:

* Changed Box 3 to follow one sentence per line convention. There are a couple minor changes in there as well.
* Removed reference to complex phenotypes and complex data from the synopsis. I also changed our reference to "large amounts of complex data" because it was somewhat imprecise.
* Fixed the Figure 1 legend per reviewer's comments.
* Revised a bunch of language in Box 1 per reviewer's comments (e.g., "held-out test set" becomes "held-out validation set").
* Added a TODO about updating the LASSO reference.

I'd like to take another look at Box 3 tomorrow, so I'm marking as a draft for now.

### CODE REVIEW

**Comment by jaybee84** [comment]
@jaclyn-taroni  Thanks for starting this draft.

I was just taking a look at the changes, and it looks like a bunch of changes that I had already made in the original (word doc) did not port correctly to the PR .. e.g. 
> Removed reference to complex phenotypes and complex data from the synopsis. I also changed our reference to "large amounts of complex data" because it was somewhat imprecise.
> Fixed the Figure 1 legend per reviewer's comments.
> Revised a bunch of language in Box 1 per reviewer's comments (e.g., "held-out test set" becomes "held-out validation set").

Thanks for going through those and fixing/updating those. Looking forward to your finalized changes.

**Comment by jaclyn-taroni** [comment]
I'm going to mark this ready for review for the benefit of running the other checks. Edit: That's not the issue, it's that I'm targeting a branch other than the default branch.

**Comment by jaclyn-taroni** [comment]
Okay, with https://github.com/jaybee84/ml-in-rd/pull/242/commits/c03e15298b10f727e93a9b65044f3cbc827a474e, I made a few tweaks to where the new box (Box 1) is referenced. Some references seemed too low context given what the sentenced around them actually said – a reader is unlikely to be familiar with the papers cited – so I've either removed something or made it clear it was an example by adding `e.g.,`.

File: content/05.model-complexity.md [hunk 1]
```diff
@@ -32,9 +32,9 @@ Regularization is often used in exploring functional role of rare variants in ra
 For example, LASSO has been used as a feature selection method for amyotrophic lateral sclerosis (ALS) gene expression data. [@doi:10.1186/s10020-023-00603-y]
 In this example, applying LASSO regularization reduced the number of genes included as features in the a machine learning model designed to classify brain tissue regions from ALS patients. 
 In the context of rare immune cell signature discovery, variations of elastic-net regression were found to outperform other regression approaches [@doi:10.1016/j.compbiomed.2015.10.008; @doi:10.1186/s12859-019-2994-z]. 
-Thus, regularization methods like LASSO or elastic-net are beneficial in ML with rare observations, and are worth exploring in the context of rare diseases.[@doi:10.1371/journal.pgen.1004754] 
+Thus, regularization methods like LASSO or elastic-net are beneficial in ML with rare observations, and are worth exploring in the context of rare diseases.[@doi:10.1371/journal.pgen.1004754] <!--TODO: replace this reference-->
```
Comment (jaybee84) [inline content/05.model-complexity.md]:
```suggestion
Thus, regularization methods like LASSO or elastic-net are beneficial in ML with rare observations, and are worth exploring in the context of rare diseases.[@doi:10.1186/s10020-023-00603-y] 
```

File: content/01.synopsis.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ## Abstract {.page_break_before}
 
 The advent of high-throughput profiling methods such as genomics and other technologies has accelerated basic research and made deep molecular characterization of patient samples routine. 
-These approaches provide a rich portrait of genes, molecular pathways, and cell types involved in complex phenotypes. 
+These approaches provide a rich portrait of genes, molecular pathways, and cell types involved in rare disease phenotypes.
```
Comment (jaybee84) [inline content/01.synopsis.md]:
```suggestion
These approaches provide a rich portrait of genes, molecular pathways, and cell types involved in disease phenotypes. 
```

File: content/01.synopsis.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
 ## Abstract {.page_break_before}
 
 The advent of high-throughput profiling methods such as genomics and other technologies has accelerated basic research and made deep molecular characterization of patient samples routine. 
-These approaches provide a rich portrait of genes, molecular pathways, and cell types involved in complex phenotypes. 
+These approaches provide a rich portrait of genes, molecular pathways, and cell types involved in rare disease phenotypes. 
 Machine learning (ML) can be a useful tool to extract disease-relevant patterns from high dimensional datasets. 
 However, depending on the complexity of the biological question, machine learning often requires a large number of samples to identify recurrent and biologically meaningful patterns. 
-Rare diseases are inherently limited in clinical cases and thus have few samples to study. 
+Rare diseases are inherently limited in clinical cases and thus have few samples to study.
 Precision medicine also presents a similar challenge, where patients with common diseases are partitioned into small subsets of patients based on particular characteristics.
```
Comment (jaybee84) [inline content/01.synopsis.md]:
```suggestion
```

File: content/06.prior-knowledge.md [hunk 4]
```diff
@@ -7,15 +7,15 @@ Knowledge graphs (KGs) which integrate related-but-different data types, provide
...
 KGs may include links (also called edges) or nodes that are specific to the rare disease of interest (e.g., an FDA approved treatment would be a specific disease-compound edge in the KG) as well as edges that are more generalized (e.g., gene-gene interactions noted in the literature for a different disease). (Figure [@fig:4]a)
 
-Rare disease researchers can repurpose general (i.e., not rare disease-specific) biological or chemical knowledge graphs to answer rare disease-based research questions [@doi:10.1142/9789811215636_0041] (e.g. Box 3b). 
-There are a variety of tactics to sift through the large amounts of complex data in knowledge graphs.
+Rare disease researchers can repurpose general (i.e., not rare disease-specific) biological or chemical knowledge graphs to answer rare disease-based research questions [@doi:10.1142/9789811215636_0041] (e.g. Box 1b). 
+There are a variety of tactics to sift through the data encoded in knowledge graphs.
```
Comment (jaybee84) [inline content/06.prior-knowledge.md]:
```suggestion
There are a variety of tactics to sift through the large amounts of data encoded in knowledge graphs.
```

File: content/05.model-complexity.md [hunk 5]
```diff
@@ -11,14 +11,14 @@ All of these contribute to low signal to noise ratio in rare disease datasets.
...
-Random forests use resampling (with replacement) based techniques to form a consensus about the important predictive features identified by the decision trees (Box 3c). [@https://doi.org/10.1023/A:1010933404324; @doi:10.1186/1472-6947-13-134]
-Additional approaches like combining random forests with resampling without replacement can generate confidence intervals for the model predictions by iteratively exposing the models to incomplete datasets, mimicking real world cases where most rare disease datasets are incomplete [@doi:10.3390/genes11020226] (Box 3d).
+Random forests use resampling (with replacement) based techniques to form a consensus about the important predictive features identified by the decision trees (e.g., Box 1c). [@https://doi.org/10.1023/A:1010933404324; @doi:10.1186/1472-6947-13-134]
+Additional approaches like combining random forests with resampling without replacement can generate confidence intervals for the model predictions by iteratively exposing the models to incomplete datasets, mimicking real world cases where most rare disease datasets are incomplete [@doi:10.3390/genes11020226].
```
Comment (jaybee84) [inline content/05.model-complexity.md]:
```suggestion
Additional approaches like combining random forests with resampling without replacement can generate confidence intervals for the model predictions (e.g. for applications like Box 1d) by iteratively exposing the models to incomplete datasets, mimicking real world cases where most rare disease datasets are incomplete [@doi:10.3390/genes11020226].
```

**Review by jaybee84 — CHANGES_REQUESTED** [review]
added some minor comments. otherwise look good to me..

**Comment by jaybee84** [comment]
@jaclyn-taroni I see that you did not make any changes to the `outlook.md` file where we refered to the excluded methods. Is it okay to assume that you did not have any additional edits there?

File: content/05.model-complexity.md (hunk 5 above)
Comment (jaclyn-taroni) [reply content/05.model-complexity.md]:
```suggestion
Additional approaches like combining random forests with resampling without replacement can generate confidence intervals for the model predictions (e.g., for applications like Box 1d) by iteratively exposing the models to incomplete datasets, mimicking real world cases where most rare disease datasets are incomplete [@doi:10.3390/genes11020226].
```

**Comment by jaclyn-taroni** [comment]
> @jaclyn-taroni I see that you did not make any changes to the `outlook.md` file where we refered to the excluded methods. Is it okay to assume that you did not have any additional edits there?

Yes, no additional edits.

**Pull request merged**